### DISCUSSION TITLE
Get container elements

### DISCUSSION THREAD

**Discussion by JS-Aibel** [zauberzeug/nicegui#799] [Ideas / Feature Requests]
According to [Clear containers](https://nicegui.io/documentation#clear_containers), the remove(element) method can remove individual elements inside a container. Is there a method that can get an element from a container?

In the example code below, a ui.row() element (container) holds two labels. Then the first label get changed.
Would be nice to do `container.get(0).set_text('test')`

```py
from nicegui import ui

with ui.row() as container:
    ui.label("label1")
    ui.label("label2")
    
container.slots['default'].children[0].set_text('test')

ui.run()
```

**Comment by falkoschindler** [comment]
Hi @JS-Aibel! Yes, that would be nice.

You can already access the label as follows:
```py
container.default_slot.children[0]
```

The `default_slot` is needed because elements can have multiple slots, each with a list of children. See https://github.com/zauberzeug/nicegui/discussions/615 for more detail. I'm not sure if we can improve on this API.

**Reply by JS-Aibel** [reply]
Ok, thanks for the explanation.
